src/maketools/makeModuleMap

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env bash
+# shellcheck disable=SC2059
+
+problems=""
+toPrint='  {"%s", "%s"}'
+#regex are scary, here's a link with the flow of the regex
+# https://regexper.com/#PLUMED_REGISTER_ACTION%5Cs*%5C%28%5Cs*%5B0-9a-zA-Z_%5D%2B%5Cs*%2C%5Cs*%22%28%5B0-9a-zA-Z_%5D%2B%29%22%5Cs*%5C%29
+regex='PLUMED_REGISTER_ACTION\s*\(\s*[0-9a-zA-Z_]+\s*,\s*"([0-9a-zA-Z_]+)"\s*\)'
```

Comment:
In the mac CI seems to ignore the action regitered like these:
```
PLUMED_REGISTER_ACTION(Dimer, "DIMER")
PLUMED_REGISTER_ACTION(DynamicReferenceRestraining, "DRR")
PLUMED_REGISTER_ACTION(EDS, "EDS")
PLUMED_REGISTER_ACTION(FuncPathGeneral, "FUNCPATHGENERAL")
PLUMED_REGISTER_ACTION(JCoupling, "JCOUPLING")
PLUMED_REGISTER_ACTION(Loss, "MAZE_LOSS")
PLUMED_REGISTER_ACTION(Memetic, "MAZE_MEMETIC_SAMPLING")
PLUMED_REGISTER_ACTION(OptimizerBias, "MAZE_OPTIMIZER_BIAS")
PLUMED_REGISTER_ACTION(Random_Acceleration_MD, "MAZE_RANDOM_ACCELERATION_MD")
PLUMED_REGISTER_ACTION(Random_Walk, "MAZE_RANDOM_WALK")
PLUMED_REGISTER_ACTION(Simulated_Annealing, "MAZE_SIMULATED_ANNEALING")
PLUMED_REGISTER_ACTION(Steered_MD, "MAZE_STEERED_MD")
PLUMED_REGISTER_ACTION(fusionPoreExpansionP, "FUSIONPOREEXPANSIONP")
PLUMED_REGISTER_ACTION(fusionPoreNucleationP, "FUSIONPORENUCLEATIONP")
PLUMED_REGISTER_ACTION(memFusionP, "MEMFUSIONP")
```
with only a space after the ","